internal/start/templates/vector_config

```diff
@@ -0,0 +1,121 @@
+data_dir: /var/lib/vector
+sources:
+  gotrue_journald:
+    type: journald
+    current_boot_only: true
+    include_units:
+      - gotrue.service
+  postgrest_journald:
+    type: journald
+    current_boot_only: true
+    include_units:
+      - postgrest.service
+transforms:
+  filter_postgrest_stats:
+    type: filter
+    inputs:
+      - postgrest_journald
+    condition: >-
+      !starts_with!(.message, "+") && !starts_with!(.message, "INFO:") && !contains!(.message, "Admin server listening")
+
+  gotrue_to_object:
+    inputs:
+      - gotrue_journald
+    type: remap
+    source: |2-
+                .project = "project_ref"
```

Comment:
Does this need to replaced by template variable?

Reply:
nope, for local dev, project ref defaults to `default`, so not necessary